backbone.stickit.js

```diff
@@ -174,7 +174,11 @@
 
             // Don't update the model if false is returned from the `updateModel` configuration.
             var currentVal = evaluateBoolean(config.updateModel, val, event, config);
-            if (currentVal) setAttr(model, modelAttr, val, options, config);
+            if (isPromise(currentVal)) {
+              currentVal.then(function(){
+                setAttr(model, modelAttr, val, options, config);
```

Comment:
I'm thinking that using promise status for it, like 'fullfill' for 'true', 'reject' for 'false', considering about Ajax, backend return error code then setAttr will not be invoked.

But... that's true that we still need some code logic even in success callback. I would change it and add test case later.